src/components/styles.js

```diff
@@ -147,6 +147,10 @@ const styles = theme => ({
     cursor: 'no-drop',
     color: '#fff!important'
   },
+  button_transparent: {
```

Comment:
If this style is to make a button match the background by being white, we may just want to adjust the opacity?

Reply:
When I adjusted the opacity of the button, the opacity of picture on the button was also affected. I removed the `backgroundColor` property instead.